tests/test_source.py

```diff
@@ -57,15 +57,16 @@ def test_source_brightness():
 
 
 def test_source_brightness_purity():
-    s = Source(brightness=0.4, purity=0.9)
+    s = Source(brightness=0.4, purity=0.9, multiphoton_model="indistinguishable")
```

Comment:
Previous versions of `Source` had `indistinguishable` as default value. Shouldn't it stay the same for backward compatibility?